packages/tasks/src/pipelines.ts

```diff
@@ -589,9 +589,15 @@ export const PIPELINE_DATA = {
 	},
 	"text-to-video": {
 		name: "Text-to-Video",
-		modality: "multimodal",
+		modality: "cv",
```

Comment:
Same comment here

Reply:
Check the discussion in https://huggingface.slack.com/archives/C01N44FJDHT/p1706796001731059?thread_ts=1706792789.298009&cid=C01N44FJDHT

This model only has one type of model input (same as text-to-speech).

Reply:
That makes sense, thank you for the link!